src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,35 @@
+package pages;
+
+import helpers.Waiter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage {
+    private final String BASE_USER = "seleniumtests10";
+    private final String PASSWORD = "060788avavav";
+
+    private final By USER_NAME = By.id("mailbox__login");
+    private final By PASSWORD_INPUT = By.id("mailbox__password");
+    private final By SIGN_IN = By.id("mailbox__auth__button");
+    private final By MAIL_RU = By.cssSelector("#cnJ1y-uP");
+
+    public void enterUserName(WebDriver driver) {
```

Comment:
Create constructor, which will take WebDriver driver as parameter and save to class field. This WebDriver field will be used in all methods of class. It will help to avoid passing WebDriver to each method of the class